openquake/hazardlib/geo/surface/gridded.py

```diff
@@ -0,0 +1,164 @@
+from openquake.hazardlib.geo.surface.base import BaseSurface
+from openquake.hazardlib.geo.mesh import Mesh
+
+
```

Comment:
No Michele it's not an abstract class but there are many methods not implemented because this is a surface based on a unstructured mesh therefore many distance types cannot be computed.